src/shared.js

```diff
@@ -1811,9 +1831,7 @@ export const getUpdatedParticipantData = async (participantData) => {
     return responseParticipant.data[0];
 }
 
-export const updateCollectionSettingData = async (biospecimenData, tubes, participantData) => {
-    participantData = await getUpdatedParticipantData(participantData);
-
+export const generateCollectionSettingData = (biospecimenData, tubes, participantData) => {
```

Comment:
I see this name change, but I don't see any calls to this function. This used to be `updateCollectionSettingData`, and it was called from the Promise.all() in `processSpecimenCollectionFormUpdates`.

Has the entire process moved to ConnectFaas?

If yes, please remove unused code. Otherwise, please reintegrate it...these can be derived client-side and sent to connectFaas.